recipes/lapack/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# bug 145
+if test `uname` = "Linux"
+then
+  sed -e 's|/CMAKE/|/cmake/|' -i CBLAS/CMakeLists.txt
+fi
+
+mkdir build
+cd build
+
+# CMAKE_INSTALL_LIBDIR="lib" suppresses CentOS default of lib64 (conda expects lib)
+
+cmake \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DCMAKE_INSTALL_LIBDIR="lib" \
+  -DBUILD_TESTING=OFF \
+  -DBUILD_SHARED_LIBS=ON \
+  -DLAPACKE=ON \
+  -DCBLAS=ON \
+  ..
+
+make
```

Comment:
Look at the recipe of libmatio-feedstock, it should fix your problem.

On Sun, May 8, 2016, 7:52 AM Julien Schueller <notifications@github.com>
wrote:

> In recipes/lapack/build.sh
> <https://github.com/conda-forge/staged-recipes/pull/531#discussion_r62427249>
> :
>
> > +
> > +mkdir build
> > +cd build
> > +
> > +# CMAKE_INSTALL_LIBDIR="lib" suppresses CentOS default of lib64 (conda expects lib)
> > +
> > +cmake \
> > +  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
> > +  -DCMAKE_INSTALL_LIBDIR="lib" \
> > +  -DBUILD_TESTING=OFF \
> > +  -DBUILD_SHARED_LIBS=ON \
> > +  -DLAPACKE=ON \
> > +  -DCBLAS=ON \
> > +  ..
> > +
> > +make
>
> @jakirkham <https://github.com/jakirkham>, I encountered a link error on
> osx, so I dont enable it for now, I just check the presence of the libraries
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/531/files/3fab5317e4620a2cb17626851271988740c60e10#r62427249>
>